mpsl/include/mpsl.h

```diff
@@ -143,6 +143,11 @@ void MPSL_IRQ_CLOCK_Handler(void);
  */
 void mpsl_low_priority_process(void);
 
+/** @brief Application needs to call this when calibration shall occur.
+ *
+ * In ncs it is designed to be called with a k_timer with period CONFIG_CLOCK_CONTROL_NRF_CALIBRATION_PERIOD
```

Comment:
We try to avoid using the NCS abbreviation in anything official.
```suggestion
 * In the nRF Connect SDK, it is designed to be called with a k_timer with a period of CONFIG_CLOCK_CONTROL_NRF_CALIBRATION_PERIOD
```